### PR TITLE
[optimizing performance] component should NOT update

### PR DESCRIPTION
**Pull request by csbun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "pub": "npm run compile && npm run build && npm publish"
   },
   "peerDependencies": {
-    "g2": "^1.2.6"
+    "g2": "^2.0.0"
```
Comment (jaredleechn) [inline package.json]: Very nice！But this line will cause a install error when using g2@1.x with g2-react, Maybe you want to upgrade the version of g2 in devDeps?

Comment (csbun) [reply package.json]: peerDependencies with `g2@^1.2.6` [DO NOT](https://travis-ci.org/antvis/g2-react/jobs/201384258#L200) pass testing on node4.x

Comment (jaredleechn) [reply package.json]: How about `>= 1.2.6 < 3`，make it work both for 1.x and 2.x

Comment (csbun) [reply package.json]: Let me try

**Comment by jaredleechn** [comment]
👍 will pub in next Tue

File: src/index.js [hunk 2]
```diff
@@ -35,6 +35,10 @@ export default function createG2(__operation) {
       }
     }
 
+    shouldComponentUpdate() {
+      return false;
+    }
```
Comment (afc163) [inline src/index.js]: I believe it is extra.

**Pull request merged**